packages/python/src/components/Value.tsx

```diff
@@ -0,0 +1,62 @@
+import { For, Indent, memo } from "@alloy-js/core";
+
+export interface ValueProps {
+  value?: unknown;
```

Comment:
`jsValue` is the convention used when providing a concrete value in JSON that is serialized into the target language.

Reply:
Fixed.